app/services/rm_api_service.rb

```diff
@@ -18,6 +18,18 @@ def request(verb, path, body = nil)
     end
   end
 
+  def request_multi(pending)
+    responses = {}
+
+    pending.each do |key, args|
+      res = request(*args)
+      responses[key] = res
+      break unless res.ok?
```

Comment:
I was thinking, do we want to just continue with the remaining requests anyway here? Maybe in the future we want to still include other request data that succeeded. The order they're added to `pending` becomes the order they're requested, so I wouldn't want any headaches from ordering the `pending` hash properly.

Reply:
I think we want to steer clear of that for now and treat it as transactional: complete success or failure.

Reply:
Alright we'll keep this, but the logic in `ok?` and `errors` for `Responses` has been updated to only return the _last_ response data since that would be the only failed one in the set given to initialize.

Reply:
Ah, I see, still transactional, but wait for all requests? I think that's fine either way. What we want to avoid is, at the endpoint level any idea of a "partial success"